src/main/Config.cpp

```diff
@@ -25,7 +25,7 @@
 
 namespace stellar
 {
-const uint32 Config::CURRENT_LEDGER_PROTOCOL_VERSION = 13;
+const uint32 Config::CURRENT_LEDGER_PROTOCOL_VERSION = 14;
```

Comment:
this commit should probably be first as you already reference protocol 14 in previous commits

Reply:
Moved to the first commit